demos/motortwit/static/js/config.js

```diff
@@ -0,0 +1,211 @@
+/* global _ angular */
+(function () {
+    'use strict';
+    var onSubmitError = function (error, form, progression, notification) {
+        // mark fields based on errors from the response
+        if (!('error_details' in error.data)) {
+            return true;
+        }
+        _.mapObject(error.data.error_details, function (error_msg, field_name) {
+            if (form[field_name]) {
+                form[field_name].$valid = false;
+            }
+            return {};
+        });
+        // stop the progress bar
+        progression.done();
+        // add a notification
+        notification.log('Some values are invalid, see details in the form',
+            {addnCls: 'humane-flatty-error'});
+        // cancel the default action (default error messages)
+        return false;
+    };
+
+
+    var app = angular.module('aiohttp_admin', ['ng-admin']);
+    app.config(['RestangularProvider', function (RestangularProvider) {
+        var token = window.localStorage.getItem('aiohttp_admin_token');
+        RestangularProvider.setDefaultHeaders({'Authorization': token});
+    }]);
+
+    app.config(['NgAdminConfigurationProvider', function (nga) {
+        var admin = nga.application('aiohttp admin demo');
+
+        admin.baseApiUrl('/admin/');
+
+        admin.errorMessage(function (response) {
+            var msg = '<p>Oops error occured with status code: ' + response.status + '</p>\n';
+
+            if ('error_details' in response.data) {
+                msg += '<code>';
+                msg += JSON.stringify(response.data.error_details, null, 2);
+                msg += '</code>';
+            }
+            return msg;
+        });
+
+        var user = nga.entity('user').identifier(nga.field('_id'));
+        var message = nga.entity('message').identifier(nga.field('_id'));
+        var follower = nga.entity('follower').identifier(nga.field('_id'));
+
+        admin
```

Comment:
Shouldn't it be
```
admin.addEntity(user)
     .addEntity(message)
     .addEntity(follower);
```
?

Reply:
sorry, those js files are not used, removing